build.sh

```diff
@@ -13,7 +13,6 @@ set -e
 set -o pipefail
 
 parse "$@"
-yarn ${YARN_OPTS}
```

Comment:
Building it with yarn allows failing faster when there's a compilation error. So, not to wait while it fails within a docker build.